mapreduce/tests/common.py

```diff
@@ -79,6 +84,28 @@
     ],
 }
 
+EXPECTED_METRICS = {
```

Comment:
It seems that not all metrics are listed here, from this list: https://github.com/DataDog/integrations-core/blob/master/mapreduce/metadata.csv

What's the rational behind a metric being covered here or not ?

Reply:
The metrics retrieved from an integration depend on the configuration file and the setup. For this setup and the configuration in `INSTANCE_INTEGRATION`, `EXPECTED_METRICS` lists 100% of the metrics.

Reply:
Hum, is it feasible to change the configuration file and the setup to cover all the metrics listed in `metadata.csv`? or there are some obstacles to that?

(maybe it's outside of scope of this PR though)

Reply:
I added 2 more metrics, but I'm having trouble getting others for now.
From my point of view, the main goal of this PR is to have a mapreduce environment where the check can run and get some metrics. I don't know if having all the metric is relevant here.